collector/aws/elbv2.go

```diff
@@ -89,11 +117,29 @@ func (el *ELBV2Manager) Detect() ([]DetectedELBV2, error) {
 	now := time.Now()
 
 	for _, instance := range instances {
+		var cloudWatchNameSpace string
+		var price float64
+		if loadBalancerConfig, found := loadBalancersConfig[*instance.Type]; found {
+			cloudWatchNameSpace = loadBalancerConfig.cloudWatchNamespace
 
-		log.WithField("name", *instance.LoadBalancerName).Debug("cheking elbV2")
-
-		price, _ := el.pricingClient.GetPrice(el.GetPricingFilterInput(), "", el.region)
+			if cloudWatchNameSpace == "" {
+				log.WithError(err).WithFields(log.Fields{
+					"name": *instance.LoadBalancerName,
+					"type": *instance.Type,
+				}).Error("Could not get CloudWatch namespace because of unknown loadbalancer type")
+				continue
+			}
 
+			log.WithField("name", *instance.LoadBalancerName).Debug("checking elbV2")
+			pricingRegionPrefix := el.pricingClient.GetRegionPrefix(el.region)
```

Comment:
Do you need the prefix login while you loop on instances? 
the value can change between instances?